CHANGELOG.md

```diff
@@ -17,18 +17,18 @@
 
 ## 0.1.3
 
-* Fix tests
-* Update dependencies
+- Fix tests
```

Comment:
Ehh, ok. This seems very opinionated. Is that a prettier thing?

Reply:
This is result of resolving conflicts. I did pay attention for that properly.